dll/win32/comctl32/imagelist.c

```diff
@@ -112,11 +112,15 @@ struct _IMAGELIST
     BOOL    color_table_set;
 
     LONG        ref;                       /* reference count */
+#ifdef __REACTOS__
+    USHORT      usVersion;                 /* hack for IL from stream. Keep version here */
+#endif
 };
 
 #define IMAGELIST_MAGIC 0x53414D58
 #ifdef __REACTOS__
 #define IMAGELIST_MAGIC_DESTROYED 0x44454144
+#define IMAGELIST_VERSION 0x101
```

Comment:
Is this a documented version number by the way? or a wine-made one?

Reply:
I dont see any documentation for this